src/kademlia/dht_tracker.cpp

```diff
@@ -192,6 +235,9 @@ namespace libtorrent { namespace dht
 		if (e || m_abort) return;
 
 		m_dht.tick();
+#if TORRENT_USE_IPV6
+		m_dht6.tick();
```

Comment:
what do you think about making m_dht an array of 1 or two elements instead? I'm not sure, but it looks like some of these calls could just be put in a loop instead of duplicated